ros2/nightly/nightly/Dockerfile

```diff
@@ -18,7 +18,18 @@ RUN apt-get update && apt-get install -q -y --no-install-recommends \
     gnupg2 \
     libssl-dev \
     lsb-release \
+    python3-flake8 \
+    python3-flake8-blind-except \
+    python3-flake8-builtins \
+    python3-flake8-class-newline \
+    python3-flake8-comprehensions \
+    python3-flake8-deprecated \
+    python3-flake8-docstrings \
+    python3-flake8-import-order \
+    python3-flake8-quotes \
```

Comment:
@mikaelarguedas , @ruffsl sorry to jump out of nowhere here. But this linter `flake8-quotes` is literally driving me crazy :roll_eyes: 

1. Is there any good reason why we are using it!?
2. Where can I find the original motivation for adding this?
3. Where can I propose to completely get rid of such linter :nerd_face: ?

Thanks in advance and sorry for the annoyance!

Reply:
Hi @nachovizzo,

This is not something that is decided on the docker images maintaining side.
This is part of the ROS 2 styleguide: https://docs.ros.org/en/humble/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html?highlight=style%20guide#id4 specifically this line:
"We pick single quotes over double quotes as long as no escaping is necessary."

I think the topic came up a couple times, I remember a thread on discourse a while back to compatible with other linters e.g. black https://discourse.ros.org/t/python-black-formatting/13277

Not sur if this is a satisfactory answer but that's a start :)

Reply:
@mikaelarguedas 🙌 Thanks! This is what I was missing , this entry point. Thanks a lot